grpc/src/main/java/com/linecorp/armeria/server/grpc/GrpcServiceBuilder.java

```diff
@@ -288,6 +292,18 @@ public GrpcServiceBuilder intercept(Iterable<? extends ServerInterceptor> interc
         return this;
     }
 
+    /**
+     * Set a custom error response mapper. This is useful to serve custom response when using unframed gRPC
+     * service.
+     * @param unframedErrorResponseMapper The function which maps {@link HttpHeaders} to a {@link HttpResponse}.
+     */
+    public GrpcServiceBuilder unframedErrorResponseMapper(
+            Function<HttpHeaders, HttpResponse> unframedErrorResponseMapper) {
```

Comment:
It would be useful to see `ServiceRequestContext` to handle an error.

How about introducing an interface to type safely take more values.
```java
public interface UnframedGrpcErrorHandler {
    static UnframedGrpcErrorHandler of() {
        // Implement the default handler using you did in `UnframedErrorResponseMappers`
        return DefaultUnframedGrpcErrorHandler.INSTANCE;
    }

    HttpResponse handler(ServiceRequestContext ctx, Status status, AggregatedHttpResponse response);
} 

class GrpcServiceBuilder {
    @Nullable
    UnframedGrpcErrorHandler unframedGrpcErrorHandler;

    public GrpcServiceBuilder unframedErrorHandler(UnframedGrpcErrorHandler unframedErrorHandler) {
       ...
    }
```
Addtionally, we can pass`UnframedGrpcErrorHandler.of()` to `UnframedGrpcService` by default when unframed is enabled.